public/css/stylesheet.css

```diff
@@ -1,216 +0,0 @@
-  height: 100%;
-  width: 60%;
-  position: absolute;
-  top: 0;
-  left: 0;
-  content: '';
-  z-index: 0;
-
-}
-
-.cohort-form-opacity-layer {
-  z-index: 10;
-  position: relative;
-}
-.form-container {
-  margin: 0 auto;
-  width: 90%;
-  min-height: 90%;
-  overflow: hidden;
-}
-
-@media(min-width: 992px) {
-  .form-container {
-    width: 60%;
-  }
-}
-
-.cohort-form-input {
-  display: block;
-  width: 70%;
-}
-
-.cohort-form-input {
-  padding: 1em;
-  margin-top: 1.5em;
-  margin-bottom: 1.8em;
-  margin: 0 auto;
-  border: 2px solid #2B2D42 ;
-  font-size: 0.8em;
-
-
-}
-
-#account{
-  text-align: center;
-    width: 188px;
-    height: 32px;
-    margin-top: 18px;
-    background-color: #2B2D42;
-    border: 1px solid #2B2D42;
-    color: white;
-    font-size: 0.8rem;
-}
-
-#week-number{
-
-    text-align: center;
-    width: 88px;
-    height: 36px;
-    margin-top: 18px;
-    background-color: #2B2D42;
-    border: none;
-    color: white;
-    font-size: 0.8rem;
-    border-left: 1px solid;
-}
-
-#add-mentor{
-  width: 50px;
-    height: 36px;
-    margin-top: 18px;
-    background-color: #2B2D42;
-    border: none;
-    color: white;
-    font-size: 0.8rem;
-    border-left: 1px solid #fff;
-}
-
-.addmentor{
-  font-size: 0;
-    margin-bottom: 50px
-}
-
-#Add-Cohorts{
-  width: 13%;
-    height: 33px;
-    margin-top: 8px;
-    background-color: #2B2D42;
-    border: none;
-    color: white;
-    font-size: 0.8rem;
-        width: 77px;
-
-}
-
-.new-inputs{
-      display: block;
-    /* margin: 15px 0 8px 0; */
-    text-align: center;
-    margin: 10px auto;
-    width: 242px;
-    height: 30px;
-    border: 1px solid #2B2D42; 
-/*    border-radius: 3px;
-*/
-}
-
-::-webkit-input-placeholder {
-  font-size: 0.8rem;
-}
-
-.cohorts-Buttons{
-  text-align: center;
-}
-
-.cohorts-Buttons button {
-    width: 13%;
-    height: 33px;
-    margin-top: 18px;
-    background-color: #2B2D42;
-    border: none;
-    color: white;
-    font-size: 0.8rem;
-}
-.addcohort-container{
-  margin-top: 55px;
-  text-align: center;
-      background: #ddd;
-    width: 453px;
-    margin: 50px auto;
-    padding: 15px;
-    border: 2px solid #2B2D42;
-}
-
-.addCohort-label {
-  display: block;
-  width: 100%;
-  color: #2B2D42;
-  font-size: 2.5rem;
-}
-
-
-.Add-Cohorts-Form{
-    /*display: flex;*/
-    flex-direction:column;
-    width: 70%;
-    margin: 0 auto;
-    /*margin: 0 auto;*/
-}
-
-
-
-th {
-    color: #2B2D42;
-    text-align: left;
-    border-bottom: 1px solid lightgrey;
-    font-size: 1.5rem;
-}
-
-#weeks-mentor-table {
-    border-collapse: collapse;
-    width: 100%;
-}
-
-.table-container {
-    width: 50%;
-    margin: 0 auto;
-    text-align: center;
-    color: #2B2D42;
-    margin-top:50px;
-}
-#weeks-mentor-table{
-      background-color: #ddd;
-    border: 2px solid #2B2D42;
-}
-tr {
-    height: 35px;
-    border: 1px solid #2B2D42;
-
-}
-td{
-      border: 1px solid #2B2D42;
-
-
-}
```

Comment:
Try to avoid using ids when you can use classes. https://stackoverflow.com/a/8279151/4224679